app/templates/index.html

```diff
@@ -33,19 +35,24 @@
                     if (!response.ok) {
                         throw new Error('Network response was not ok');
                     }
-                    return response.blob();
-                })
-                .then(blob => {
-                    const url = URL.createObjectURL(blob);
-                    const imgElement = document.getElementById('scannedImage');
-                    imgElement.src = url;
-                    imgElement.style.display = 'block';
+                    return response.json();
                 })
+                .then(data => {
+                    if (data.detected_text) {
+                        const textDisplayElement = document.getElementById('scannedText');
+                        textDisplayElement.textContent = data.detected_text;
+                        textDisplayElement.style.display = 'block';
+                    } else {
+                        console.error('No text detected or returned');
+                        alert('Nie wykryto tekstu lub wystąpił błąd.');
```

Comment:
chyba lepiej jak zrobimy tak, że wyświetlamy zdjęcie i niżej text, czy coś takieo. Albo jak sie wciśniej przycisk scan, to zapytania na 2 endpointy, ten twój do tekstu, mój do wyświetlania zdjęcia. Ale to chyba trzeba jakiś meeting zrobić, żeby to ustalić bardziej.

Reply:
Ten tekst to tak na chwilę, żebyście mogli zobaczyć jak działa. Jak skończę pisać regex'a to wrzucę tam tabelkę z produktami i cenami.

Reply:
Nie musimy chyba patrzeć jak działa bo nie mamy klucza do api na razie. Twój endpoint może zostać jako testowy, ale szczerze to bardziej widze to w ten sposób, że skanujemy zdjęcie w tym moim, zapisujemy text do bazy. A jeśli chcemy zobaczyć  zapisany text to wczytujemy go z bazy. Chyba tak będzie lepiej.

@bartek3060 co myślisz

Reply:
Zrobione